docs-chef-io/content/server/upgrades.md

```diff
@@ -218,6 +235,23 @@ If you are running a Chef Infra Server release before 12.3.0, please contact Che
    reindexdb: reindexing database "template1"
    ```
 
+1. Run `ANALYZE` on the PostgreSQL database.This process gathers statistics for the query planner to create the most efficient query execution paths. Accurate statistics will help the planner to choose the most appropriate query plan, and thereby improve the speed of query processing.
+    ```bash
+       sudo su - opscode-pgsql
+       /opt/opscode/embedded/bin/vacuumdb --all --analyze-only
+    ```
+
+   You should then see output like:
+   
+    ```bash
+       vacuumdb: vacuuming database "bifrost"
+       vacuumdb: vacuuming database "oc_id"
+       vacuumdb: vacuuming database "opscode-pgsql"
+       vacuumdb: vacuuming database "opscode_chef"
+       vacuumdb: vacuuming database "postgres"
+       vacuumdb: vacuuming database "template1"
```

Comment:
We have suggested the vacuum full at the beginning of the process, but depending on the size of the data compared to the system that can also be done just before running analyze, optionally.

Reply:
yes, the suggestion here would be that vacuum at the start is recommended to make the process faster, but the customers can also do that after the upgrade.

Reply:
Have added them